### PR TITLE
Implement Guzzle v6

### PR DESCRIPTION
**Pull request by Echron**
Implementation of latest Guzzle client
+ Add reporting example

### CODE REVIEW

**Comment by arendjantetteroo** [comment]
Thanks! Will try to look into this soon.

Is it backwards compatible? So for users of this client only the dependencies change? Or do we need a new major version so we dont break current implementations?

File: src/AJT/Toggl/ReportsClient.php [hunk 1]
```diff
@@ -23,38 +20,25 @@ class ReportsClient extends Client
...
+        $clientConfig = self::getClientConfig($config);
 
-        $client->setDescription($description);
+        $guzzleClient = new Client($clientConfig);
 
-        $client->setDefaultHeaders(array(
-            "Content-type" => "application/json",
-        ));
+        if (isset($config['apiVersion']) && $config['apiVersion'] !== 'v2') {
+            throw new \Exception('Only v8 is supported at this time');
```
Comment (arendjantetteroo) [inline src/AJT/Toggl/ReportsClient.php]: Only v2 for reporting is supported at this time?

**Comment by arendjantetteroo** [comment]
Thanks @Echron , tried it out, the examples seem to work great still. 

Will add a new major version

**Comment by Echron** [comment]
I refactored it to be backwards compatible, i wrote some simple tests which all worked after the refactoring. But since there are no unit tests i'm not 100% sure. I tested with all the existing examples and added one for reporting.

File: src/AJT/Toggl/ReportsClient.php (hunk 1 above)
Comment (Echron) [inline src/AJT/Toggl/ReportsClient.php]: Ah yes, copy paste error. I'm not sure why to keep 2 clients who do almost the same except for the different definition file. But i'm not gonna change that ;)

**Comment by arendjantetteroo** [comment]
Released 1.0 with your work. Thanks again.

**Pull request merged**